### ISSUE TITLE
Error: Cannot read properties of null (reading 'login')

### ISSUE THREAD

**Issue by keiguard** [mikesprague/teams-incoming-webhook-action#75]
Hello, first would like to thank you for this action. It's been very useful. Up until today this github action has been working flawlessly for me. But I have started to get the below error without changing anything in my GitHub Actions yaml file:
`Error: Cannot read properties of null (reading 'login')`

The applicable code for the GitHub actions file is below:
```
      - name: Deploy Success Notification
        if: ${{ success() }}
        uses: mikesprague/teams-incoming-webhook-action@v1
        with:
          github-token: ${{ secrets.PAT_DEV }}
          webhook-url: ${{ secrets.MS_TEAMS_WEBHOOK_URI }}
          deploy-card: true
          title: "Deployment Successful"
          color: "success"
          timezone: "Africa/Johannesburg"
```

Do you know if this error is being caused by something in my code or not? I have been making recent changes so it is possible.

The error logs for this specific action are below:
```
Run mikesprague/teams-incoming-webhook-action@v1
  with:
    github-token: ***
    webhook-url: ***
    deploy-card: true
    title: Deployment Successful
    color: success
    timezone: Africa/Johannesburg
  env:
    AZURE_WEBAPP_NAME: opt-pro-dev
    REPO: opt-pro/optimum-docker
    AZURE_HTTP_USER_AGENT: 
TypeError: Cannot read properties of null (reading 'login')
    at populateCard (file:///home/runner/work/_actions/mikesprague/teams-incoming-webhook-action/v1/dist/webpack:/teams-incoming-webhook-action/src/lib/cards/deploy.js:[4](https://github.com/opt-pro/optimum-docker/actions/runs/3337814799/jobs/5524680424#step:5:4)9:1)
    at file:///home/runner/work/_actions/mikesprague/teams-incoming-webhook-action/v1/webpack:/teams-incoming-webhook-action/src/index.js:76:1
    at processTicksAndRejections (node:internal/process/task_queues:96:[5](https://github.com/opt-pro/optimum-docker/actions/runs/3337814799/jobs/5524680424#step:5:5))
```

**Comment by mikesprague** [comment]
@keiguard Thanks for reporting the this! 

A similar issue was previously reported in #68 (albeit with less information so I was not able to properly look into it).

With the additional detail I have determined that the `login` prop for author is only being populated sometimes and that I can't count on it being there. It's not clear what causes this but I will be implementing a fix that doesn't use the data if it's not there.

I expect the fix to go out sometime today and you should get a notification once this has been resolved.

**Comment by mikesprague** [comment]
This should be fixed in the latest release (via commit b76c30a).

Please reopen with any additional relevant details if you have any additional problems.

**Comment by keiguard** [comment]
@mikesprague thank you very much! For further describing the issue as well as the fast fix.
